TTHAnalysis/python/analyzers/MT2Skimmer.py

```diff
@@ -29,12 +29,17 @@ def process(self, event):
         self.counters.counter('events').inc('all events')
 
         
-        ret = False 
-        
+        nElectrons10 = sum([l.pt() > 10 and abs(l.pdgId()) == 11 for l in event.selectedLeptons])
+        nMuons10 = sum([l.pt() > 10 and abs(l.pdgId()) == 13 for l in event.selectedLeptons])
+        nPFLep5LowMT = event.nPFLep5LowMT
+        nPFHad10LowMT = event.nPFHad10LowMT
+        nLep = nElectrons10 + nMuons10 + nPFLep5LowMT + nPFHad10LowMT
         ht = event.htJetXj10l5t
         nJet30 = sum([j.pt() > 30 for j in event.cleanJets])
         mt2= event.mt2_Xj
         met_pt = event.met.pt()
+        diffMetMht = event.diffMetMht
```

Comment:
One uses the object event.mhtJet40j10l5tvec (jet pt threshold of 40 instead of 30?) while the other uses event.mhtJetXj10l5tvec. 
https://github.com/MT2Analysis/cmgtools-lite/blob/80X/TTHAnalysis/python/analyzers/ttHCoreEventAnalyzer.py#L387-L397

For the current data production I checked that there is no practical difference, so no need for another production.